api/v1alpha1/envoygateway_types.go

```diff
@@ -483,15 +483,22 @@ type XDSTranslatorHooks struct {
 // ExtensionService defines the configuration for connecting to a registered extension service.
 type ExtensionService struct {
 	// Host define the extension service hostname.
-	Host string `json:"host"`
+	// Deprecated: use the backend attribute instead
+	//
+	// +optional
+	Host string `json:"host,omitempty"`
 
 	// Port defines the port the extension service is exposed on.
+	// Deprecated: use the backend attribute instead
 	//
 	// +optional
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:default=80
 	Port int32 `json:"port,omitempty"`
 
+	// Backend points to where the extension server can be found.
+	Backend BackendEndpoint `json:"backend"`
```

Comment:
should this be an embedded field ? since TLS is on this level

Reply:
Done